src/test/resources/xoom-cluster.properties

```diff
@@ -59,7 +59,10 @@ cluster.live.node.timeout = 20000
 cluster.quorum.timeout = 60000
 
 # currently all active nodes must be listed as seed nodes
-cluster.seedNodes = node1,node2,node3,test-grid
+cluster.nodes = node1,node2,node3,test-grid
```

Comment:
Is it still a requirement to list all possible nodes, or can the cluster scale linearly to virtually any number of nodes? (The latter is the intention for true elastic scale.)

Reply:
The communication between cluster nodes is happening on two types of channels: application and operational. Application channels were not changed in this PR. They require the list of all configured (static) cluster nodes. Also, `ConfirmingDistributor` requires the list of all configured (static) cluster nodes. `LocalRegistry` has been changed and it contains now the list of live nodes.

In order to achieve complete elastic scalability we need to adjust as well application channels and `ConfirmingDistributor` to use live nodes provided by `LocalRegistry` (most likely in a subsequent PR).